src/components/starReview/starReview.tsx

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+import styled from "@emotion/styled";
+import Icon from "@/src/ui-library/components/icons/svg";
+import {theme} from "@/src/ui-library/theme";
+
+interface StarOption {
+    size: "small" | "medium" | "large";
+    amount: 1 | 2 | 3 | 4 | 5;
+    variant: "Review" | "Reviews";
+}
+
+const StarContainer = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-content: center;
+  column-gap: ${(props) => (props.variant === "Review" ? "unset" : "10px")};
+`;
+
+const StarReview: React.FC<StarOption> = ({size, amount, variant}) => {
+    const sizeToIconSize = {
+        small: 12,
+        medium: 20,
+        large: 30,
+    };
+
+    const iconSize = sizeToIconSize[size] || 12;
```

Comment:
타입스크립트 사용하고 계시기 때문에 safefy 하게 사용 가능해보입니다. 방어코드 제거 해도 되겠네요 ㅎㅎ

```suggestion
    const iconSize = sizeToIconSize[size]
```

Reply:
감사합니다. 확인 후 수정 해두었습니다.